### PR TITLE
Fix mixed content warnings

### PR DESCRIPTION
**Pull request by Bisaloo**
I noticed these warnings while netlify was preparing the preview of a blog post. Mixed content warnings signal that some resources (images, script, iframes...) are loaded over HTTP while the main page is loaded over HTTPS (more info: https://support.mozilla.org/en-US/kb/mixed-content-blocking-firefox).

I found two cases of mixed content:

- passive mixed content with images. This is not a big issue from a security point of view but it still creates a warning in most browsers (warning) that may be worrying to some users (or teach them to ignore this kind of warning which may be even worse).

- active mixed content with iframes. This is more problematic since active mixed content could pose serious security issues and is therefore blocked by most browsers. This means the content in the iframe won't be displayed.

### CODE REVIEW

File: content/blog/2013-11-04-data-to-cartodb.md [hunk 1]
```diff
@@ -103,7 +103,7 @@ lapply(rows, function(x) cartodb.row.insert(name = "pumamap", columns = list("na
 
 After the upload is finished, I had to make sure the table was georeferenced, and played with settings to suit my style. And then I made a visualization from the `pumamap` dataset and made it public. And that's it!  You can [find the map](https://cdb.io/1fbvgCG), and it can be embedded:
 
-<iframe width='100%' height='400' frameborder='0' src='http://recology.cartodb.com/viz/76a5598e-444b-11e3-8a8c-5404a6a69006/embed_map?title=false&description=false&search=false&shareable=false&cartodb_logo=true&layer_selector=false&legends=false&scrollwheel=true&sublayer_options=1&sql=&zoom=3&center_lat=37.61423141542417&center_lon=-100.72265625'></iframe>
+<iframe width='100%' height='400' frameborder='0' src='https://recology.cartodb.com/viz/76a5598e-444b-11e3-8a8c-5404a6a69006/embed_map?title=false&description=false&search=false&shareable=false&cartodb_logo=true&layer_selector=false&legends=false&scrollwheel=true&sublayer_options=1&sql=&zoom=3&center_lat=37.61423141542417&center_lon=-100.72265625'></iframe>
```
Comment (Bisaloo) [inline content/blog/2013-11-04-data-to-cartodb.md:106]: This link seems broken over both HTTP and HTTPS because of an issue with google maps.

Comment (maelle) [reply content/blog/2013-11-04-data-to-cartodb.md:106]: @sckott Could you help fix this?

Comment (sckott) [reply content/blog/2013-11-04-data-to-cartodb.md:106]: I changed the base map layer to a different one. is it fixed now?

Comment (Bisaloo) [reply content/blog/2013-11-04-data-to-cartodb.md:106]: I think so. Here's what I get:

image

Comment (sckott) [reply content/blog/2013-11-04-data-to-cartodb.md:106]: looks right

**Comment by maelle** [comment]
Thanks again @Bisaloo! I'll try to pay more attention to these warnings in the future!

**Comment by Bisaloo** [comment]
Haha, you're welcome. Working with HTTPS is how I got into coding so I tend be to extra aware of this kind of stuff ^^.

**Comment by maelle** [comment]
Thanks for spreading your awareness! :star2:

**Pull request merged**